ocaml/util/version.mli

```diff
@@ -1,12 +0,0 @@
-val git_id : string
-val hostname : string
-val date : string
-val product_version : unit -> string
-val product_version_text : unit -> string
-val product_version_text_short : unit -> string
-val platform_name: string
-val platform_version: string
-val product_brand : unit -> string
-val build_number : unit -> string
-val xapi_version_major : int
-val xapi_version_minor : int
```

Comment:
Just pushed https://github.com/xapi-project/xen-api/pull/2683/commits/c05d62dc0753ff347f382e34232d75b0e27f1a4c which has reinstated (and modified) version.mli